bin/opm

```diff
@@ -1406,7 +1527,7 @@ sub install_target ($$) {
         err "no library files found in $dist_dir/.\n";
     }
 
-    if ($remove_old) {
+    if ($package_ctx->{remove_old}) {
```

Comment:
We should avoid such immediate hash table look ups since when there is a typo in the hash table key here, it is hard to detect and debug.